ci/integration.sh

```diff
@@ -85,25 +85,27 @@ function show_head {
     echo "Head commit of ${INTEGRATION}: $head"
 }
 
+tempdir=$(mktemp -d)
+
 case ${INTEGRATION} in
     cargo)
-        git clone --depth=1 https://github.com/rust-lang/${INTEGRATION}.git
-        cd ${INTEGRATION}
+        git clone --depth=1 https://github.com/rust-lang/${INTEGRATION}.git ${tempdir}
```

Comment:
Using a temp dir so we don’t pick up rustfmt’s rustfmt.toml. This shouldn’t matter for this PR bc of the feature flag, but will long term if the PR lands.